src/main/kotlin/socket/player/PlayerEvents.kt

```diff
@@ -53,4 +53,7 @@ data class MessageData(val message: String, val sound: String?, val playerIds: L
 data class PlayerXPGainData(val playerId: String, val gain: Int, val reason: String, val notify: Boolean, val multiplier: Float?)
 
 @Serializable
-data class DisconnectPlayerData(val playerId: String, val reason: String)
\ No newline at end of file
+data class DisconnectPlayerData(val playerId: String, val reason: String)
+
+@Serializable
+data class PlayerAchievementData(val player: SimplePlayer, val achievement: String, val completionTime: Long)
```

Comment:
could you clarify in the parameter name whether the `achievement` string refers to its ID, name, etc. i.e. `achievementId`, `achievementName`...